python/hail/methods/impex.py

```diff
@@ -1714,6 +1714,59 @@ def import_vcf(path,
     return MatrixTable(jmt)
 
 
+@typecheck(metadata_file=str,
+           call_fields=oneof(str, sequenceof(str)),
+           reference_genome=nullable(reference_genome_type),
+           entry_field_array_elements_required=bool,
+           tar=nullable(str))
+def import_genomicsdb(metadata_file,
+                      call_fields=[],
+                      reference_genome='default',
+                      entry_field_array_elements_required=True,
+                      tar=None) -> MatrixTable:
+    """Import GenomicsDB database.
+
+    .. include:: _templates/experimental.rst
+
+    Parameters
+    ----------
+    metadata_file : :obj:`str`
+        Path to GenomicsDB JSON metadata file.
+    call_fields : :obj:`list` of :obj:`str`
```

Comment:
This would be a reasonable change.  I think the current setup is due to special treatment to GT in the VCF spec (it is clearly specified as a call, it must be first if present, etc.)  Can we leave it for a separate PR and I'll just make sure import_{vcf, genomicsdb} agree for now?

Reply:
OK